common/src/db/test.rs

```diff
@@ -11,14 +12,36 @@ pub struct TrustifyContext {
     tempdir: Option<TempDir>,
 }
 
+/// collect database information for the external test database.
+///
+/// **NOTE:** This may panic in case where the [`Database`] arguments cannot be parsed.
+#[allow(clippy::expect_used)]
+fn external_test_db() -> Database {
+    #[derive(clap::Parser)]
+    struct Cli {
```

Comment:
This seems like such an abuse of clap to me. The computer running the program won't care, but the humans maintaining this code might.